components/input.css

```diff
@@ -0,0 +1,59 @@
+.input-section {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2rem;
+}
+.input-section h2 {
+  font-size: 48px;
+  margin: 2rem 0rem 2rem 0rem;
+}
+.input-section h4 {
+  font-size: 24px;
```

Comment:
Suggestion: don't mix rem/px unit. Try to use rem unit.

Reply:
Thanks for the suggestion. I have changed that.